samples/ConsoleApp.SelfBootstrap/Program.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Diagnostics;
+using System.Net.Http;
+using System.Threading.Tasks;
+using OpenTelemetry;
+using OpenTelemetry.Trace;
+
+namespace ConsoleApp.SelfBootstrap
+{
+    internal class Program
+    {
+        private static readonly ActivitySource MyActivitySource = new ActivitySource("OpenTelemetry.AutoInstrumentation.ConsoleApp");
+        private static TracerProvider _tracerProvider;
+
+        private static async Task<int> Main()
+        {
+            var builder = Sdk
+                .CreateTracerProviderBuilder()
+                .AddHttpClientInstrumentation()
+                .AddSqlClientInstrumentation()
+                .SetSampler(new AlwaysOnSampler())
+                .AddSource("OpenTelemetry.AutoInstrumentation.*")
```

Comment:
nit: add a comment indicating why this is needed.

Reply:
Also, it would be good to add an "application-specific" source as @pjanotti noted in the previous comment.

Reply:
Done